compression_algorithms/huffmann/huffmann.go

```diff
@@ -0,0 +1,275 @@
+package main
+
+import (
+    "container/heap"
+    "fmt"
+    "io/ioutil"
```

Comment:
https://pkg.go.dev/io/ioutil

This library seems to be deprecated.

Reply:
Updated with os library which has methods to read file